idaes/models/properties/modular_properties/base/generic_property.py

```diff
@@ -1100,6 +1125,7 @@ def define_metadata(cls, obj):
                 "mw": {"method": "_mw"},
                 "mw_comp": {"method": "_mw_comp"},
                 "mw_phase": {"method": "_mw_phase"},
+                "number_prandtl_phase": {"method": "_number_prandtl_phase"},
```

Comment:
I would name this `Prandlt_number_phase` - I don't think we have a standard for this yet (but could be wrong).